rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -1997,14 +1998,16 @@ rocalVideoFileSource(
             THROW("Sequence length passed should be bigger than 0")
         // Set video loader flag in master_graph
         context->master_graph->set_video_loader_flag();
+        if ((source_path != nullptr) && (source_path[0] != '\0') && file_names_list.size() != 0)
+            THROW("file_root and filenames are mutually exclusive")
```

Comment:
We should check the condition if both are not set.

Reply:
Done